admin-cli/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id 'bio.terra.catalog.java-spring-conventions'
+
```

Comment:
Nit - extra whitespace.

Reply:
It's totally arbitrary but I've been splitting up the versioned and non-versioned plugin references with a whitespace in-between. Although now that you point this out, I realize I should make this a non-versioned reference since it's used in multiple projects now.